docs/minio-client-complete-guide.md

```diff
@@ -562,6 +563,13 @@ encryptedobject:    14 B / 14 B  ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 ```
 Notice that two different aliases myminio1 and myminio2 are used for the same endpoint to provide the old secretkey and the newly rotated key.
 
+*Example: Copy file and assign Cache-Control header for it*
```

Comment:
Copy a javascript file to object storage and assign `Cache-Control` header to the uploaded object.

Reply:
updated